### DISCUSSION TITLE
$('#otp_target').otpdesigner('set', ""); 							$('.otp-content').html("");

### DISCUSSION THREAD

**Discussion by smartland007** [HichemTab-tech/OTP-designer-jquery#5] [Ideas]
Actually to clear the OTP (ex: when the code is false), we need to do the following, where each input have the class otp-content:
$('.otp-content').html("");

I was assuming to use the following:
$('#otp_target').otpdesigner('set', "");

Is it possible to have a dedicated event to clear/empty the code ?

**Comment by HichemTab-tech** [comment]
yes it is possible thanks for bringing this to attention.
it will be resolved soon !.

**Comment by HichemTab-tech** [comment]
Hi @smartland007 ,

We are glad to inform you that we have recently added a new feature to address the scenario you described.

With the latest update, you can now use the 'clear' method to easily clear the OTP code directly from the plugin. This new method is designed to provide a more streamlined approach for clearing the OTP, making it more intuitive for developers like you.

Instead of using the jQuery approach with `$('.otp-content').html("");`, you can now use the dedicated 'clear' method like this:

```
$('#otp_target').otpdesigner('clear');
```


This method will automatically clear the OTP input fields.

Please feel free to let us know if you have any further questions or suggestions. Your feedback is invaluable in helping us improve the library.

Thank you for using OTP-designer-jquery!

Best regards,
@HichemTab-tech
